hms-app/src/components/NewIdea.tsx

```diff
@@ -1,15 +1,42 @@
+import {
+  Textarea,
+  Group,
+  Button,
+  useMantineTheme,
+  CheckboxGroup,
+  Checkbox,
+  createStyles,
+  Card,
+} from '@mantine/core'
 import React, { useState } from 'react'
 
+const useStyles = createStyles((theme) => ({
+  card: {
+    backgroundColor:
+      theme.colorScheme === 'dark' ? theme.colors.dark[7] : theme.white,
+  },
+  section: {
+    borderBottom: `1px solid ${
+      theme.colorScheme === 'dark' ? theme.colors.dark[4] : theme.colors.gray[3]
+    }`,
+    paddingLeft: theme.spacing.md,
+    paddingRight: theme.spacing.md,
+    paddingBottom: theme.spacing.md,
+    textTransform: 'uppercase',
+    fontSize: theme.fontSizes.md,
+    fontWeight: 500,
+  },
+}))
+
 function NewIdea() {
-  //TODO automaticly add ID and owner for a idea
+  const { classes } = useStyles()
+  const [skills, setSkills] = useState<string[]>([])
   const [ideaText, setIdeaText] = useState({
     title: '',
     description: '',
     reason: '',
     problem: '',
     goal: '',
-    skills: '',
-    aws_needed: '',
   })
 
   function handleChange(event: { target: { name: any; value: any } }) {
```

Comment:
try to avoid any type, as it is basically avoiding type safety. Better go with:
`function handleChange(event: React.ChangeEvent<HTMLTextAreaElement>) {`

Reply:
done